src/layouts/index.js

```diff
@@ -12,24 +12,33 @@ import styles from './index.module.css'
 
 addLocaleData(localeData)
 
-const Layout = ({ children }) => (
-  <IntlProvider locale="<%= locale %>" messages={ messages }>
-    <div className={ styles.app }>
-      <Helmet
-        title="JS IPFS"
-        meta={ [
-          { name: 'description', content: 'JS IPFS website' },
-          { name: 'keywords', content: 'gatsby, ipfs' }
-        ] } />
+class Layout extends Component {
+  componentDidMount () {
+    require('intersection-observer')
```

Comment:
Do you need this here?

Reply:
Leave a comment explaining this option

Reply:
Yes, we need this here because we need to be sure that component is mounted and `window` exists. Otherwise, we will get an error stating that `window is not defined`.